docs/fundamentals/code-analysis/quality-rules/ca2109.md

```diff
@@ -24,6 +24,11 @@ ms.author: gewarren
 
 A public or protected event-handling method was detected.
 
+> [!NOTE]
+> This rule has been deprecated. It last shipped with Microsoft.CodeAnalysis.NetAnalyzers 7.0.0 NuGet package and the .NET 7 SDK.
+>
+> The rule was removed because the threat that the analyzer warned about (an untrusted intermediary hooking a privileged event handler to a privileged event invoker) hasn't existed since .NET Framework 4.5.
```

Comment:
> an untrusted intermediary hooking a *privileged* event handler to a *privileged* event invoker

@GrabYourPitchforks Can you confirm if there is a typo here? I'm thinking it's about *unprivileged* invokers hooking into a *privileged* events? I could definitely be wrong as I'm not familiar with this rule.